build.gradle.kts

```diff
@@ -175,6 +175,20 @@ dependencies {
         exclude(group = "jakarta.xml.bind", module = "jakarta.xml.bind-api")
         exclude(group = "jakarta.validation", module = "jakarta.validation-api")
     }
+    implementation("com.monitorjbl:xlsx-streamer:2.1.0") {
```

Comment:
This dependency can be removed as it was replaced with `excel-streaming-reader`. See line 173 in this file.

Reply:
Removed in 3094c7271a67d1b79ea297c9859b1487eb43ae61